dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/authentication/ApiTokenResourceTest.java

```diff
@@ -0,0 +1,66 @@
+package com.dotcms.rest.api.v1.authentication;
+
+import com.dotcms.datagen.CompanyDataGen;
+import com.dotcms.datagen.UserDataGen;
+import com.dotcms.rest.InitDataObject;
+import com.dotcms.rest.RestUtilTest;
+import com.dotcms.rest.WebResource;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.dotmarketing.util.UUIDGenerator;
+import com.liferay.portal.ejb.CompanyPool;
+import com.liferay.portal.model.Company;
+import com.liferay.portal.model.User;
+import com.liferay.portal.util.WebKeys;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.ws.rs.core.Response;
+
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.anyObject;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class ApiTokenResourceTest {
+
+    @BeforeClass
+    public static void prepare() throws Exception{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5164094)